Guide/tailwindcss.markdown

```diff
@@ -114,7 +114,7 @@ form label {
 
 We need to add a new build command for starting a tailwind build process to our `Makefile`. For that append this to the `Makefile` in your project:
 
-```makefile
```

Comment:
I guess we a better solution would be to fix the css instead :) Likely we just need to add a `, .language-makefile` to this css rule here: https://github.com/digitallyinduced/ihp/blob/master/Guide/layout.html#L194